cookbook/integrations/external_services/databricks/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM databricksruntime/standard:11.3-LTS
+LABEL org.opencontainers.image.source=https://github.com/flyteorg/flytesnacks
```

Comment:
can you include an example docker build command?  as a comment i mean.  here or in the docs somewhere (or did i miss it)?

Reply:
added it